modules/virt-enabling-usb-host-passthrough.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * virt/advanced_vm_management/virt-configuring-usb-host-passthrough.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-enabling-usb-host-passthrough_{context}"]
+= Enabling USB host passthrough
+
+You can enable USB host passthrough at the cluster level.
+
+You specify a resource name and USB device name for each device you want to add and then assign to a virtual machine (VM). You can allocate more than one device, each of which is known as a `selector` in the HyperConverged (HCO) custom resource (CR), to a single resource name. If you have multiple, identical USB devices on the cluster, you can choose to allocate a VM to a specific device.
```

Comment:
> You specify a resource name and USB device name for each device you want to add and then assign _[the names? the device?]_ to a virtual machine (VM). 

> You can allocate more than one device, each of which is known as a `selector` _[I get what this means, but might be rewritten to satisfy the repo guidelines, e.g. "Each device is represented by a `selector` object/property...]_ in the HyperConverged (HCO) custom resource (CR), to a single resource name.

Reply:
@maxwelldb, I don't understand the second comment. Is there a source or style guide link you can provide?

Reply:
For posterity:

ISG: "When you refer to a command, parameter, or option in running text, specify the word command, parameter, or option after the name. Command, parameter, and option names are not translated, so without the identifying noun, users might be confused about what the name represents. Alternatively, refer to a command, parameter, or option by using a lowercase descriptive phrase, such as one that includes an adjective and a noun."

Same guidance, essentially, for programming elements.

In general, if something is backticked in running text, it has to function as an adjective for translation's sake.